### ISSUE TITLE
Training code for multi-view controlnet 

### ISSUE THREAD

**Issue by anita-hu** [IGLICT/SketchDream#6]
Hi, awesome work! Are you planning to release the code for training the multi-view controlnet model?
Thanks!

**Comment by ty625911724** [comment]
Thanks a lot for your attention. We may release the training code in future, but do not sure the timeline.

**Comment by anita-hu** [comment]
For training images, did you use random grayscale background similar to original MVDream?

**Comment by ty625911724** [comment]
Yes, we render objects into png format and attach grayscale background
